package.json

```diff
@@ -196,4 +195,4 @@
     "type": "opencollective",
     "url": "https://opencollective.com/styled-components"
   }
-}
+}
```

Comment:
This should not be changed I think

Reply:
Yeah it's ok, prettier keeps doing it but it doesn't break anything